.github/workflows/test_suite.yml

```diff
@@ -5,26 +5,24 @@ on:
   push:
     branches:
       - main
+    paths:
+      - '**.py'
+      - '**.msh'
+      - '**.geo'
+      - '.github/workflows/test_suite.yml'
```

Comment:
Yeah it's probably fine not to test after such merges, given that we test every week anyway.

Reply:
Sorry, not sure if I understood. You'd like to remove these triggers from `push: main`?

Reply:
Apologies - I wasn't that clear. My comment was saying what you've introduced is a good idea.